server/test/unit/BraveAlerterConfiguratorTest/getReturnMessageToRespondedByPhoneNumberTest.js

```diff
@@ -6,83 +6,122 @@ const { before, describe, it } = require('mocha')
-      'en',
-      CHATBOT_STATE.STARTED,
-      CHATBOT_STATE.WAITING_FOR_CATEGORY,
-      ['Cat0', 'Cat1'],
-    )
-
-    expect(returnMessage).to.equal('Once you have responded, please reply with the number that best describes the incident:\n1 - Cat0\n2 - Cat1\n')
-  })
-
-  it('should get message when WAITING_FOR_REPLY => WAITING_FOR_CATEGORY', () => {
-    const returnMessage = this.alertStateMachine.getReturnMessageToRespondedByPhoneNumber(
-      'en',
-      CHATBOT_STATE.WAITING_FOR_REPLY,
-      CHATBOT_STATE.WAITING_FOR_CATEGORY,
-      ['Cat0', 'Cat1'],
-    )
-
-    expect(returnMessage).to.equal('Once you have responded, please reply with the number that best describes the incident:\n1 - Cat0\n2 - Cat1\n')
-  })
-
-  it('should get message when WAITING_FOR_CATEGORY => WAITING_FOR_CATEGORY', () => {
-    const returnMessage = this.alertStateMachine.getReturnMessageToRespondedByPhoneNumber(
-      'en',
-      CHATBOT_STATE.WAITING_FOR_CATEGORY,
-      CHATBOT_STATE.WAITING_FOR_CATEGORY,
-      ['Cat0', 'Cat1'],
-    )
-
-    expect(returnMessage).to.equal("Sorry, the incident type wasn't recognized. Please try again.")
-  })
-
-  it('should get message when WAITING_FOR_CATEGORY => COMPLETED', () => {
-    const returnMessage = this.alertStateMachine.getReturnMessageToRespondedByPhoneNumber(
-      'en',
-      CHATBOT_STATE.WAITING_FOR_CATEGORY,
-      CHATBOT_STATE.COMPLETED,
-      ['Cat0', 'Cat1'],
-    )
-
-    expect(returnMessage).to.equal(`Thank you! This session is now complete. (You don't need to respond to this message.)`)
-  })
-
-  it('should get message when COMPLETED => COMPLETED', () => {
-    const returnMessage = this.alertStateMachine.getReturnMessageToRespondedByPhoneNumber('en', CHATBOT_STATE.COMPLETED, CHATBOT_STATE.COMPLETED, [
-      'Cat0',
-      'Cat1',
-    ])
-
-    expect(returnMessage).to.equal('Thank you')
-  })
-
-  it('should get default message if given something funky', () => {
-    const returnMessage = this.alertStateMachine.getReturnMessageToRespondedByPhoneNumber('en', 'something funky', CHATBOT_STATE.COMPLETED, [
-      'Cat0',
-      'Cat1',
-    ])
-
-    expect(returnMessage).to.equal('Error: No active session found')
+  languages.forEach(language => {
+    it('should get message when STARTED => WAITING_FOR_REPLY', () => {
+      const returnMessage = this.alertStateMachine.getReturnMessageToRespondedByPhoneNumber(
+        language,
+        CHATBOT_STATE.STARTED,
+        CHATBOT_STATE.WAITING_FOR_REPLY,
+        ['Cat0', 'Cat1'],
+      )
+
+      if (language === 'en') {
+        expect(returnMessage).to.equal(
+          `Once you have responded, please reply with the number that best describes the incident:\n1 - Cat0\n2 - Cat1\n`,
+        )
+      } else {
+        expect(returnMessage).to.equal(`Una vez que haya contestado, responda con el número que mejor describa el incidente:\n1 - Cat0\n2 - Cat1\n`)
+      }
```

Comment:
Same comment as above in server/test/unit/BraveAlerterConfiguratorTest/getReturnMessageToOtherResponderPhoneNumbersTest.js